htdocs/core/modules/import/import_csv.modules.php

```diff
@@ -809,7 +809,7 @@ public function import_insert($arrayrecord, $array_match_file_to_database, $obji
 										break;
 									}
 									$classinstance = new $class($this->db);
-									$res = call_user_func_array(array($classinstance, $method), array(&$arrayrecord, $listfields, ($key - 1)));
+									$res = call_user_func_array(array($classinstance, $method), array(&$arrayrecord, $listfields, $key));
```

Comment:
This looks strange. Everywhere in the rest of the code of this file, we use $key -1.
Exemple: line 578, line 587, line 746, line 764, ...
so the key to get the value inside the &arrayrecord is ($key - 1).

Reply:
@eldy I change as you said